docs/V1/Partner/answer_questionnaires.md

```diff
@@ -0,0 +1,74 @@
+Répondre à un questionnaire
+=====================================================
+
+>**:warning: toutes les urls fournies dans cette documentation commencent par /api-saas/test/v1/. Elles ne sont utilisées que pour les tests de fonctionnement (aucun enregistrement), une fois le code implémentée, veuillez utiliser une url de la forme /api-saas/v1/**
+
+Pré requis
```

Comment:
It's "préquis"! http://www.larousse.fr/dictionnaires/francais/pr%C3%A9requis/63646